quickstart.md

```diff
@@ -164,250 +166,51 @@ enclave-cc      enclave-cc      9m55s
 ### Platform Setup
 
 While the operator deploys all the required binaries and artifacts and sets up runtime classes that use them,
-certain platforms may require additional configuration to enable confidential computing. For example, the host
-kernel and firmware might need to be configured.
-See the [guides](./guides) for more information.
+certain platforms may require additional configuration to enable confidential computing. For example, a specific 
+host kernel or firmware may need to be utilized. The guides section has further instructions on how to apply
+individual TEE environments.
 
-# Running a workload
+## Using CoCo
 
-## Creating a sample CoCo workload
+The CoCo operator environment has been deployed and setup!
```

Comment:
I think this belongs a couple lines up .. at the end of the install section before this section begins